package.json

```diff
@@ -3,9 +3,18 @@
   "version": "0.1.0",
   "license": "",
   "dependencies": {
-    "deeplearn": "~0.5.0",
-    "deeplearn-knn-image-classifier": "^0.3.0",
-    "pre-commit": "^1.2.2"
+    "deeplearn": "^0.3.12",
+    "deeplearn-knn-image-classifier": "^0.2.0",
+    "@tensorflow-models/knn-classifier": "^0.2.2",
+    "@tensorflow-models/mobilenet": "^0.2.2",
+    "@tensorflow/tfjs": "^0.13.0",
+    "autoless": "^0.1.7",
```

Comment:
Are these new dependencies actually required? Also, deeplearn should be removed

Reply:
Or are you not using budo anymore?

Reply:
still using budo it's still in the dev dependencies... removed deep learn and other unnecessary dependencies

Reply:
okay should be good now... main changes are removed deeplearn, added tfjs, and added babel-polyfill to the dev dependencies